MentorBook.Web/Controllers/UserFriendshipController.cs

```diff
@@ -0,0 +1,34 @@
+using MentorBook.Business;
+using MentorBook.Data.Models;
+using MentorBook.Web.Models.QueryModels;
+using MentorBook.Web.Models.ViewModels;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MentorBook.Web.Controllers
+{
+
+    [Route("api/[controller]")]
+    [ApiController]
+ 
+    public class UserFriendshipController : ControllerBase
+    {
+            private readonly IFriendshipService _friendshipController;
+            public UserFriendshipController(IFriendshipService friendshipController)
+            {
+                _friendshipController = friendshipController;
+            }
+
+            [HttpGet("GetUserFriendsCount")]
+            public ActionResult<List<int>> GetNumber(int userID)
```

Comment:
I think you should return int, not List<int>

Reply:
Yeah I tried that too but every time I tried to work with int it'd tell me it requires list because the Query requires List